cirq/google/devices/known_devices.py

```diff
@@ -201,7 +201,7 @@ def _json_dict_(self):
     'cz': 50_000,
     'xy': 20_000,
     'z': 0,
-    'meas': 1_000_000,
+    'meas': 4_000_000,  # 1000 ms for readout, 3000ns for "ring down"
```

Comment:
I think you're mixing up microseconds and milliseconds here. Did you mean `us`?

Reply:
I actually meant nano-seconds.  I got it correct later in the same sentence at least...